source/docs/software/pathplanning/trajectory-tutorial/creating-drive-subsystem.rst

```diff
@@ -166,15 +166,15 @@ First, we create a member instance of the ``DifferentialDriveOdometry`` class:
 
   .. group-tab:: Java
```

Comment:
This is missing the configuration of the odometry class

Reply:
As with the other comment, the content is the same between latest and this PR. If we want to add more code to the sample, I'd say that's out of scope for this PR.